### ISSUE TITLE
Hackernews proof has failed

### ISSUE THREAD

**Issue by arnabwahid** [keybase/keybase-issues#1874]
Here is the link to my HN profile. The post is there.

https://news.ycombinator.com/user?id=arnabwahid

**Comment by maxtaco** [comment]
I think because you account is new you're not showing up in their [API](https://hacker-news.firebaseio.com/v0/user/arnabwahid)

**Comment by mrmch** [comment]
I'm having this problem as well, I have >= 2 karma. Does the keybase snippet need to replace the entire bio?

**Comment by maxtaco** [comment]
Sometimes firebase is very slow.  Check your firebase API page.

**Comment by sigmarelax** [comment]
My hackernews proof is also failing. 
https://news.ycombinator.com/user?id=sigmar
https://hacker-news.firebaseio.com/v0/user/sigmar

I guess it isn't a keybase bug, as the API page doesn't seem to have my "about" indexed. It has been about 30 hours. Is the API really that slow, or am I missing something?

**Comment by maxtaco** [comment]
I don't think you are missing anything. That API is really slow and flakey.
We tried to scrape HN directory but were black holed.  So this is as good
as we can do for now.  Thanks!

**Comment by oconnor663** [comment]
@mrmch when I `keybase id mrmch` I actually see your HN proof passing. Maybe it fixed itself recently? I've been in touch with support at HN, so it's possible they're making it through a backlog of fixes.

@arnabwahid and @sigmarelax, I've mentioned your usernames to HN support, hopefully they'll get to you shortly.

**Comment by mrmch** [comment]
@oconnor663 odd, this is how it appears to me on my profile, but `keybase id mrmch` shows it verified as well.

![screen shot 2016-02-08 at 1 47 11 pm](https://cloud.githubusercontent.com/assets/522045/12900314/81a03ee8-ce6a-11e5-8a2a-709e092862c5.png)

**Comment by oconnor663** [comment]
Does clicking "retry" fix your profile on the website? @maxtaco this is expected, right?

**Comment by maxtaco** [comment]
Yes. Clearly the client doesn't take the server's word for it, so it's not totally a surprise that there's a confusion between the two.  But I wouldn't have fully expected this current situation.  @mrmch what happens when you click "retry"? Thanks...
